README.md

```diff
@@ -1188,6 +1191,43 @@ All options from BDD tests do also apply here.
 
 **NOTE:** Performance tests should be run without concurrency.
 
+#### List of test tags
+
+| Tag name           | Tag meaning                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------- |
+| @smoke             | Smoke tests verifying basic functionality                                          |
+| @performance       | Performance tests                                                                  |
+| @olm               | OLM integration tests                                                              |
+| @travelagency      | Travel agency tests                                                                |
```

Comment:
Should not this tag be a feature rather than a tag?

Reply:
The travel agency tests are located in separate feature file.
Question is whether we need a specific tag for it?

Reply:
Travel agency like onboarding (annotation @discovery) are the complex examples that we have.
I would like to be able to launch them separately.
If you think of another name for the tag, then we can change it :)

Reply:
Well, there is a slight difference as `discovery` test a Kogito capability of discovering other services using tags. Travel agency doesn't test any specific capability, it is "just" a complex scenario.

Though it may have sense to keep the tag there, for the case of checking Travel agency functionality.